### PR TITLE
Device compliance report datasource

### PR DESCRIPTION
**Pull request by gibarrama**
image
image
image

* Added Device Compliance Report Datasource

### CODE REVIEW

**Comment by terraform-providers-svc** [comment]
Can one of the admins verify this patch?

File: examples/data-sources/ome_compliance_report/data-source.tf [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+//Get the details of the device compliance report
+data "ome_compliance_report" "compliance_report_data" {
+  // This would get the baseline id and return it and its details
+  baseline_id = 6
```
Comment (doriac11) [inline examples/data-sources/ome_compliance_report/data-source.tf:17]: Changes this back to baseline_name = "example_name"

File: examples/data-sources/ome_compliance_report/provider.tf [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+terraform {
+  required_providers {
+    ome = {
+      source = "registry.terraform.io/dell/ome"
+    }
+  }
+}
+
+provider "ome" {
+  username = "admin"
```
Comment (doriac11) [inline examples/data-sources/ome_compliance_report/provider.tf:23]: Remove these actual values please just use empty strings ""

File: examples/data-sources/ome_compliance_report/provider.tf [hunk 3]
```diff
@@ -0,0 +1,27 @@
+/*
+Copyright (c) 2023 Dell Inc., or its subsidiaries. All Rights Reserved.
```
Comment (doriac11) [inline examples/data-sources/ome_compliance_report/provider.tf:2]: Delete this duplicate file

**Review by doriac11 — APPROVED** [review]

**Pull request closed without merging**